sdk/tests/deqp/framework/common/tcuFloat.js

```diff
@@ -60,7 +60,7 @@ tcuFloat.FloatDescription = function(exponentBits, mantissaBits, exponentBias, f
 tcuFloat.FloatDescription.prototype.zero = function(sign) {
     return tcuFloat.newDeFloatFromParameters(
         deMath.shiftLeft((sign > 0 ? 0 : 1), (this.ExponentBits + this.MantissaBits)),
-        new tcuFloat.FloatDescription(this.ExponentBits, this.MantissaBits, this.ExponentBias, this.Flags)
+        this
```

Comment:
As far as I can tell there's no purpose to creating new copies of the descriptor. The descriptor properties are never altered, so these seem to be unnecessary copies. I'm guessing it's that was directly ported from the C++ code where it likely made more sense.